code/game/objects/items.dm

```diff
@@ -85,8 +85,8 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
 	if(ismob(loc))
 		var/mob/m = loc
 		m.unEquip(src, 1)
-	for(var/X in actions)
-		qdel(X)
+	QDEL_LIST(actions)
+	master = null

```

Comment:
This is why assemblies/grenades don't GC; it's so weird that this is a var on all items when only assemblies and assemblies holders use it.